src/Lib/package.cpp

```diff
@@ -34,17 +42,67 @@ const std::string Package::GetFullyQualifiedName(void) {
   return Package::GetName() + " " + Package::GetVersion();
 }
 
+void Package::assignDataDir(void) {
+  std::vector<std::string> possible{".", "..", DATADIR};
+
+  char *npDataEnv = getenv("NETPANZER_DATADIR");
+  if (npDataEnv != NULL) possible.insert(possible.begin(), npDataEnv);
+  char *npAppDir = getenv("APPDIR");
+  if (npAppDir != NULL) possible.insert(possible.end(), npAppDir);
+  int length = possible.size();
+
+  int i = 0;
+  while (i < length) {
+    std::ifstream file(possible[i] + "/maps");
+    if (file.good()) {
+      std::filesystem::path p = possible[i];
+      Package::setDataDir(std::filesystem::absolute(p).parent_path().parent_path());
```

Comment:
Meh, this isn't the right way to find the absolute path yet. But the result is so far the same. The binary works in the source root, but not the build dir.